packages/ag-charts-community-examples/src/charts-overview/examples/simple-line/main.ts

```diff
@@ -39,6 +39,7 @@ const options: AgChartOptions = {
             marker: {
                 stroke: '#01c185',
                 fill: '#01c185',
+                strokeWidth: 0
```

Comment:
Ideally this would be:

```diff
-stroke: '#01c185,
fill: '#01c185'
```

However then the highlighting cases fail since highlighting applies the `highlightSeries.strokeWidth` to both the line and the marker. And the marker would then have the default colour, not the custom colours seen here.